unreleased.rst

```diff
@@ -31,5 +31,6 @@ HEAD — ongoing
   instead of the contract id.
 - [DAML Triggers] ``getTemplates`` has been renamed to ``getContracts`` to describe its behavior more accurately.
   ``getTemplates`` still exists as a compatiblity helper but it is deprecated and will be removed in a future SDK release.
-- [JSON API - Experimental] Fix to support Archive choice. See issue #3219.
```

Comment:
oops... why are you removing? I don't think it was released yet.

Reply:
See 4230fe271d80bd91c89359a10d5f75b4f9170c17